Adding basic sources configuration (ping source) (#4328)
knative/docs: docs/admin/eventing/sources-configuration.md

> # Sources Configurations

Reviewer: ```suggestion
# Configure event source defaults
```

> Each Knative Source can be configured depending on how it is generating events. This document contains details about how to configure the available Sources that can be configured.

Reviewer: ```suggestion
This topic describes how to configure defaults for Knative event sources. You can configure event sources depending on how they generate events.
```

> ## Ping Source Configurations

Reviewer 2: Ping Source -> PingSource

> ## Ping Source Configurations

Reviewer: ```suggestion
## Configure defaults for PingSource
```

> A [Ping Source](../../developer/eventing/sources/ping-source/README.md) is an event source that produces events with a fixed payload on a specified cron schedule. You can find how to create new Ping Source and the available parameters [here](../../developer/eventing/sources/ping-source/reference.md).

Reviewer: ```suggestion
PingSource is an event source that produces events with a fixed payload on a specified cron schedule. For how to create a new PingSource, see [Creating a PingSource object](../../developer/eventing/sources/ping-source/README.md).
For the available parameters, see [PingSource reference](../../developer/eventing/sources/ping-source/reference.md).
```

> Besides the parameters that can be configured in the `PingSource` resources, there is a global `ConfigMap` called `config-ping-defaults`.
> 
> This `ConfigMap` allows you to change the maximun amount of data that the `PingSource` will add to the produced CloudEvents.

Reviewer: ```suggestion
In addition to the parameters that you can configure in the PingSource resource, there is a global ConfigMap called `config-ping-defaults`.
This ConfigMap allows you to change the maximum amount of data that the PingSource adds to the CloudEvents it produces.
```

> The `dataMaxSize` parameter allows you to set the max number of bytes allowed to be sent for message excluding any base64 decoding. The default value (-1) is no limit set for data.

Reviewer: ```suggestion
The `dataMaxSize` parameter allows you to set the maximum number of bytes allowed to be sent for a message excluding any base64 decoding. The default value, `-1`, sets no limit for data.
```

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: config-ping-defaults
  namespace: knative-eventing
data:
  dataMaxSize: -1
```

> You can edit this `ConfigMap` by running:

Reviewer: ```suggestion
You can edit this ConfigMap by running the command:
```
